evalml/objectives/standard_metrics.py

```diff
@@ -359,7 +360,14 @@ def objective_function(self, y_true, y_predicted, X=None):
         if (y_true == 0).any():
             raise ValueError("Mean Absolute Percentage Error cannot be used when "
                              "targets contain the value 0.")
-        return (np.abs((y_true - y_predicted) / y_true)).mean() * 100
+        # Use numpy ufuncs to avoid errors with custom indices
+        if isinstance(y_true, pd.Series):
+            y_true = y_true.values
+        if isinstance(y_predicted, pd.Series):
+            y_predicted = y_predicted.values
+        difference = np.subtract(y_true, y_predicted)
+        scaled_difference = np.divide(difference, y_true)
+        return np.abs(scaled_difference).mean() * 100
```

Comment:
@freddyaboulton very cool!

The call to `.values` converts to a np type -- is that necessary here, or will the first call to one of the np ufuncs do that conversion for you?

Reply:
I ended up removing the ufuncs cause calling `.values` is all we need!